HesoyamHospital/GraphicEditorTests/Integration/ScheduleSpecialistAppointmentTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Backend;
+using Backend.Model.UserModel;
+using Backend.Repository.MySQLRepository.HospitalManagementRepository;
+using Backend.Service.HospitalManagementService;
+using Shouldly;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace GraphicEditorTests.Integration
+{
+    public class ScheduleSpecialistAppointmentTest
+    {
+        [Fact]
+        public void Find_room_with_equipment()
```

Comment:
This test seems like it is dependent on data within the database, so it would fail on deployment.
Since we are not deploying your application, you don't have to change it.